WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportFormMetadataProvider.swift

```diff
@@ -55,7 +59,8 @@ class SupportFormMetadataProvider {
             Constants.platformTag,
             site.isWordPressComStore ? Constants.wpComTag : nil,
             site.plan.isNotEmpty ? site.plan : nil,
-            stores.isAuthenticatedWithoutWPCom ? Constants.authenticatedWithApplicationPasswordTag : nil
+            stores.isAuthenticatedWithoutWPCom ? Constants.authenticatedWithApplicationPasswordTag : nil,
+            addProductFromImageEligibilityChecker.isEligible() ? ABTest.addProductFromImage.rawValue : nil
```

Comment:
❓ Do we need to use the AB test name in the tag? That looks confusing to me with the date postfix. Should we change to something simpler and easier to recognize? Like `ai_product_details_from_image`. I don't think it's necessary to specify the platform either. If Android is gonna adopt the same feature, we can share the same Zendesk tag. WDYT?

Reply:
Good point. 

- I thought the idea is to keep this tag only until we run this ABTest. i.e. Remove this tag when we stop the experiment and deploy a variation.
-  Now I understand that it will make sense to keep this even after we remove the A/B test.

I have renamed the tag in ef95888bf6cf73dc5d2fb8657dc1fb098f4a54ab